src/Config/CS_SDK.props

```diff
@@ -4,21 +4,20 @@
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
     <VisualStudioVersion Condition="'$(VisualStudioVersion)' == ''">16.0</VisualStudioVersion>
     <TargetFramework>net48</TargetFramework>
+    <RuntimeIdentifier>win-x64</RuntimeIdentifier>
```

Comment:
why do we need this, for what nuget package?

Reply:
This is actually not needed right now. But will be needed one we start to compile for multiple OS's
RuntimeIdentifier is mostly used when you publish a nuget package.

Reply:
I can remove it ...and will add it when it makes sense